internal/controllers/machine/machine_controller_phases.go

```diff
@@ -245,13 +245,19 @@ func (r *Reconciler) reconcileInfrastructure(ctx context.Context, cluster *clust
 		return ctrl.Result{}, err
 	}
 	if infraReconcileResult.RequeueAfter > 0 {
```

Comment:
I rebased off of main and there weren't any conflicts. The only thing is that we can't test it in e2e tests for this PR since the CAPD changes are in the other PR.

Reply:
Thank you!